cstar/roms/component.py

```diff
@@ -471,3 +442,80 @@ def post_run(self) -> None:
                     cwd=out_path,
                     shell=True,
                 )
+
+
+class ROMSDiscretization(Discretization):
+    """
+    An implementation of the Discretization class for ROMS.
+
+
+    Additional attributes:
+    ----------------------
+    n_procs_x,n_procs_y: int
+        The number of processes following the x and y directions, for running in parallel
```

Comment:
Formatting and clarity nitpick

```suggestion
    n_procs_x,
        The number of parallel processors over which to subdivide the x axis of the domain.
    n_procs_y: int
        The number of parallel processors over which to subdivide the y axis of the domain.
```

Reply:
Addressed in b69800f70ca3a583ffdde906d663d847df97404b